charts/node-analyzer/templates/daemonset-node-analyzer.yaml

```diff
@@ -63,7 +63,21 @@ spec:
             path: /
 {{- end }}
         - name: tmp-vol
+          {{- if .Values.nodeAnalyzer.runtimeScanner.storageClassName }}
+          ephemeral:
+            volumeClaimTemplate:
+              metadata:
+                labels:
+                  type: {{ .Values.nodeAnalyzer.runtimeScanner.storageClassName }}
+              spec:
+                accessModes: [ "ReadWriteOnce" ]
+                storageClassName: {{ .Values.nodeAnalyzer.runtimeScanner.storageClassName }}
+                resources:
+                  requests:
```

Comment:
we're mapping limits to requests and we're not including limits as key here. Is that made on purpose?

Reply:
@guidobonomi i have updated this. It should be referencing the proper req/limits now. Please feel free to re-review